superset-frontend/src/explore/components/RowCountLabel.tsx

```diff
@@ -31,11 +31,20 @@ const propTypes = {
   loading: PropTypes.bool,
 };
 
+type RowCountLabelProps = {
+  rowcount?: number;
+  limit?: number;
+  rows?: string;
+  suffix?: string;
+  loading?: boolean;
+};
+
 const defaultProps = {
```

Comment:
To make sure that `defaultProps` types follow are compatible with `RowCountLabelProps`, we should either add `& typeof defaultProps` to type definition or... get rid of defaultProps and just do `const { rowcount, limit, suffix = t('rows'), loading } = props;`. 
According to https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/default_props/, the second solution is preferred

Reply:
You're right, when removing `defaultProps` while removing `propTypes` I changed it this way right away, according to the current convention.